tests/secrets.bats

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bats
+
+load helpers
+
+function setup() {
+    rm -rf $TESTSDIR/containers
+    rm -rf $TESTSDIR/rhel
+    mkdir $TESTSDIR/containers
+    touch $TESTSDIR/mounts.conf
+    MOUNTS_PATH=$TESTSDIR/containers/mounts.conf
+    echo "$TESTSDIR/rhel/secrets:/run/secrets" > $MOUNTS_PATH
+
+    mkdir $TESTSDIR/rhel
+    mkdir $TESTSDIR/rhel/secrets
+    touch $TESTSDIR/rhel/secrets/test.txt
+    echo "Testing secrets mounts. I am mounted!" > $TESTSDIR/rhel/secrets/test.txt
+}
+
+@test "bind secrets mounts to container" {
+    if ! which runc ; then
+		skip
+	fi
+	runc --version
+	cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json alpine)
+	buildah config "$cid" --default-mounts-file "$MOUNTS_PATH"
+	run buildah --debug=false run $cid ls /run
+    echo "$output"
+	[ "$status" -eq 0 ]
+	mounts="$output"
+    grep "secrets" <<< "$mounts"
+     echo "$output"
+    [ "$status" -eq 0 ]
```

Comment:
The values of `$output` and `$status` are only changed by `run`, which hasn't been called since the last time they were printed and checked.  Without the `run` error, this test will fail if `grep` doesn't find a match, since it will exit with non-zero status in that case.

Reply:
fixed.